spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before :each do
+    @user1 = User.create!(name: 'Ron', photo: 'https://unsplash.com/photos/1.jpg', bio: 'Teacher from Mexico.',
+                          posts_counter: 3)
+    @user2 = User.create!(name: 'Harry', photo: 'https://unsplash.com/photos/2.jpg', bio: 'Teacher from England.',
+                          posts_counter: 2)
+    @user3 = User.create!(name: 'Tom', photo: 'https://unsplash.com/photos/3.jpg', bio: 'Teacher from Scotland.',
+                          posts_counter: 0)
+  end
+
+  describe 'index page' do
```

Comment:
Great job team 🤝
- As stated in the project requirements, kindly add tests that assert you can see the number of posts each user has written.